### DISCUSSION TITLE
Can’t connect Rowy to an existing firebase project

### DISCUSSION THREAD

**Discussion by pippoberto** [AntlerVC/firetable#1168] [Support / Q&A] [answered]
I can’t connect my Rowy account to an existing firebase project because it says "This project has already been connected to Rowy. Please select a different one."
Previously I connected another Rowy account to the same project but then I had to change email so I deleted the Rowy account and I removed the permissions from the firebase rules, from the google console and from the Firebase users but it still says that the project is connected to the old account. 
What could I do? 
Thank you
<img width="649" alt="3662B09D-ADA5-4334-8F93-FBBAF5D53660" src="https://user-images.githubusercontent.com/64846189/226737145-82236009-0b84-4cf5-bb80-ae2ff3536c18.png">

**Comment by BeeBombshell** [comment, marked as answer]
Hey @pippoberto, I saw your post on the Discord Support Forum. I see you have followed all the steps on the Docs page: https://docs.rowy.io/faqs/deleted-project and we have successfully reconfigured your project on our end. Could you try reconnecting your project to Rowy now?

**Reply by ABESYH** [reply]
please reconfigured mine to... i have the same exact issue ...

**Comment by bofeiw** [comment]
We added a self-service "disconnect" button, to connect an existing project, you can disconnect it during project setup.


![image](https://github.com/rowyio/rowy/assets/34177142/bc5eba74-2ad6-4e88-a43c-019b5764f354)
